packages/web-react/src/components/Tooltip/demo/TooltipDismissibleViaJS.tsx

```diff
@@ -1,29 +1,30 @@
 import React, { useEffect, useState } from 'react';
 import { Button } from '../../Button';
-import TooltipWrapper from '../TooltipWrapper';
-import Tooltip from '../Tooltip';
+import { Tooltip, TooltipPopover, TooltipTrigger } from '..';
 
 const TooltipDismissibleViaJS = () => {
   const localStorageValue = localStorage.getItem('my-tooltip-react') === 'true';
   const [open, setOpen] = useState(localStorageValue);
 
-  const openHandler = () => setOpen(true);
-  const closeHandler = () => setOpen(false);
-
   useEffect(() => {
     localStorage.setItem('my-tooltip-react', open.toString());
   }, [open]);
 
   return (
-    <>
+    <div className="docs-Stack docs-Stack-stretch">
```

Comment:
already part of the wrapper, afaik. 
```suggestion
    <>
```